contracts/interfaces/RevokeAndPublish.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity 0.4.19;
+import '../EthereumClaimsRegistry.sol';
+
+
+/// @title Revoke and Publish - an interface for publishing data and 
+///        rotating access to publish new data
+contract RevokeAndPublish {
+
+    mapping(address => address) public manager;
+    EthereumClaimsRegistry registry = EthereumClaimsRegistry(0xAcA1BCd8D0f5A9BFC95aFF331Da4c250CD9ac2Da);
+
+    function revokeAndPublish(address genesis, bytes32 key, bytes32 data, address newManager) public {
+        publish(genesis, key, data);
```

Comment:
The lack of apparent access control here is confusing IMO. I’d rather see a modifier doing the access control than having this done implicitly in `publish`.

Reply:
I agree. I assume that you see that it works anyway, the reason I didn't do a modifier was because that would be executed twice -> consume more gas. Not 100% sure on that though. Let me investigate.